gst/nnstreamer/tensor_common.c

```diff
@@ -400,6 +409,10 @@ gst_tensor_config_from_video_info (GstTensorConfig * config,
   config->info.dimension[2] = height;
   config->info.dimension[3] = 1; /** Supposed 1 frame in tensor, change this if tensor contains N frames. */
 
+  for (i = 4; i < NNS_TENSOR_RANK_LIMIT; i++) {
```

Comment:
:no_entry:   This code causes dead_error_condition. 
It means that it can not be always reached this statment: `config->info.dimension[i] = ...`
Please fix it with another PR.